packages/connector-apple/src/utils.ts

```diff
@@ -0,0 +1,5 @@
+import { customAlphabet } from 'nanoid';
+
+const alphabet = '0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz';
+// FIXME @darcy: Temporary use this workaround, this change has been made in @logto/core-kit but has not been published yet.
+export const buildIdGenerator = (size: number) => customAlphabet(alphabet, size);
```

Comment:
nit: can we move generators to `connector-kit`

Reply:
you can publish separately, no need for this workaround i think

Reply:
Updated and tested locally.